### PR TITLE
libmp3lame 3.100

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **libmp3lame/3.100**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
what is this `continuous-integration/jenkins/branch` build which failed ?

**Comment by danimtb** [comment]
@ericLemanissier no idea what happened there, but don't worry

File: recipes/libmp3lame/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+
+class LibMP3LameConan(ConanFile):
+    name = "libmp3lame"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL."
+    homepage = "http://lame.sourceforge.net"
+    topics = ("conan", "libmp3lame", "multimedia", "audio", "mp3", "decoder", "encoding", "decoding")
+    license = "LGPL-2.0"
+    exports_sources = ["6410.patch", "6416.patch", "android.patch"]
```
Comment (danimtb) [inline recipes/libmp3lame/all/conanfile.py]: Could you clarify the purpose of those patches?

Comment (ericLemanissier) [reply recipes/libmp3lame/all/conanfile.py]: according to https://github.com/bincrafters/conan-libmp3lame/commit/a79d68930ba60bc9e7bf9a874af491854d3f60b7 6410.patch and 6516.patch are needed for mingw
android.patch fixes this issue: https://github.com/bincrafters/community/issues/588

Comment (SSE4) [reply recipes/libmp3lame/all/conanfile.py]: @ericLemanissier these numbers refer to SVN commits from their repository (however, not released yet...):
https://sourceforge.net/p/lame/svn/6410/
https://sourceforge.net/p/lame/svn/6416
both commits are about some MinGW specific issues:
https://sourceforge.net/p/lame/bugs/487/
https://sourceforge.net/p/lame/bugs/75/

**Pull request merged**